source/App.cpp

```diff
@@ -168,12 +168,16 @@ void App::loadModels() {
 	}
 
 	// Add all the unqiue targets to this list
-	Table<String, Any> toBuild;
+	Table<String, Any> targetsToBuild;
+	Table<String, String> explosionsToBuild;
+	Table<String, float> explosionScales;
 	for (TargetConfig target : experimentConfig.targets) {
```

Comment:
I'm noticing that in this loop, if multiple targets have the same `id` then they'll overwrite each other in the table. Does the Any reader report duplicate target `id`s as an error?

Reply:
Targets are/have historically been assumed to be unique on `id` (this is the only field by which they are referenced in the `trials` array within the session list). 

Currently we do not throw an exception if a target ID is duplicated, but rather just use the first instance in the array. We can certainly add an exception (at experiment config load time) for repeat target entries.

Reply:
I've created [an issue](https://github.com/NVlabs/abstract-fps/issues/109) to track this. The solution will likely be developed in a new branch.